bookwyrm/templatetags/layout.py

```diff
@@ -10,3 +11,9 @@ def get_lang():
     """get current language, strip to the first two letters"""
     language = utils.translation.get_language()
     return language[0 : language.find("-")]
+
+
+@register.simple_tag(takes_context=False)
+def get_path():
```

Comment:
After some personal tests, this tag doesn't work 🤔

Reply:
would it be easier to add `DOMAIN` to data available in every template via the context processor? https://github.com/bookwyrm-social/bookwyrm/blob/main/bookwyrm/context_processors.py